doc/ledger3.texi

```diff
@@ -6453,7 +6459,7 @@ been executed, such as periodic gathering.
 Redirect the output of ledger to the file defined in @file{FILE}.
 
 @item --pager @var{FILE}
-Specify the pager program to use.
+Direct output to @var{FILE} pager program.
```

Comment:
How about using `PROGRAM` or something similar instead of `FILE` to better describe the argument to the `--pager` option